test/test_satencoder.cpp

```diff
@@ -17,9 +17,8 @@ TEST_F(SatEncoderTest, CheckEqualWhenEqualRandomCircuits) {
     qc::CircuitOptimizer::flattenOperations(circOne);
     auto circTwo = circOne.clone();
 
-    SatEncoder               sat_encoder;
-    std::vector<std::string> inputs;
-    bool                     result = sat_encoder.testEqual(circOne, circTwo, inputs);
+    SatEncoder sat_encoder;
```

Comment:
Try to use consistent naming throughout the project.
```suggestion
    SatEncoder satEncoder;
```
Usually we use `camelCase`.
This is not the only place in the project where there is a mismatch in conventions, so please consider taking care of that.